multires_image/CMakeLists.txt

```diff
@@ -163,7 +163,26 @@ install(DIRECTORY launch
 
 pluginlib_export_plugin_description_file(mapviz mapviz_plugins.xml)
 
-ament_export_dependencies(${RUNTIME_DEPS} Qt)
+set(RUNTIME_DEPS
+  ament_cmake
+  Boost
+  cv_bridge
+  gps_msgs
+  mapviz
+  OpenCV
+  OpenGL
+  pluginlib
+  Qt5Core
+  Qt5Gui
+  Qt5OpenGL
+  Qt5Widgets
+  rclcpp
+  swri_math_util
+  swri_transform_util
+  tf2
+)
+
+ament_export_dependencies(${RUNTIME_DEPS})
 ament_export_include_directories(include)
 ament_export_libraries(${PROJECT_NAME})
-ament_package()
+ament_package(CONFIG_EXTRAS cmake/export_components.cmake)
```

Comment:
This seems like a really weird way to export this. Is this actually the recommended approach?

Reply:
After research it is, but this is very silly.